src/pollution_model/run_pollution_model.py

```diff
@@ -0,0 +1,474 @@
+"""
+This script takes in appropriate datasets as input and runs the MEDUSA2.0 model to calculate
+TSS (total suspended solids), TCu (total copper), DCu  (dissolved copper), TZn (total zinc), and DZn (dissolved zinc).
+
+DOI of the model paper: https://doi.org/10.3390/w12040969
+"""
+
+import logging
+import math
+from enum import StrEnum
+
+import geopandas as gpd
+from sqlalchemy.engine import Engine
+from sqlalchemy.ext.declarative import declarative_base
+
+from src.digitaltwin import setup_environment
+from src.digitaltwin.utils import LogLevel, setup_logging, get_catchment_area
+
+log = logging.getLogger(__name__)
+
+Base = declarative_base()
+
+
+# Enum strings are assigned as they are described in the original paper
+class SurfaceType(StrEnum):
+    CONCRETE_ROOF = "Cr"
+    COPPER_ROOF = "Cu"
+    GALVANISED_ROOF = "Gv"
+    ASPHALT_ROAD = "Rd"
+    CAR_PARK = "CrP"  # CarParks are classified the same as roads
+
+
+def compute_tss_roof_road(surface_area,
+                          antecedent_dry_days,
+                          average_rain_intensity,
+                          event_duration,
+                          surface_type):
+    """
+    Calculates the total suspended solids (TSS) for a surface, given the following parameters;
+
+    Parameters
+    ----------
+    surface_area: float
+        surface area of the given surface type
+    antecedent_dry_days: float
+        length of antecedent dry period (days)

```

Comment:
I think in this case, `antecedent_dry_days` is a `float`, because that's the form of the example data in the paper. But I agree with the points you raise.

Reply:
> I think in this case, `antecedent_dry_days` is a `float`, because that's the form of the example data in the paper. But I agree with the points you raise.

Agreed :)